src/main/java/com/google/cloud/teleport/templates/common/JdbcConverters.java

```diff
@@ -99,64 +99,11 @@
     ValueProvider<String> getDisabledAlgorithms();
 
     void setDisabledAlgorithms(ValueProvider<String> disabledAlgorithms);
-  }
 
-  /** Factory method for {@link ResultSetToTableRow}. */
-  public static JdbcIO.RowMapper<TableRow> getResultSetToTableRow() {
-    return new ResultSetToTableRow();
-  }
+    @Description("schmea")
+
+    ValueProvider<String> getSchema();
 
-  /**
-   * {@link JdbcIO.RowMapper} implementation to convert Jdbc ResultSet rows to UTF-8 encoded JSONs.
-   */
-  private static class ResultSetToTableRow implements JdbcIO.RowMapper<TableRow> {
-
-    static SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd");
-    static DateTimeFormatter datetimeFormatter =
-        DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss.SSSSSS");
-    static SimpleDateFormat timestampFormatter =
-        new SimpleDateFormat("yyyy-MM-dd hh:mm:ss.SSSSSSXXX");
-
-    @Override
-    public TableRow mapRow(ResultSet resultSet) throws Exception {
-
-      ResultSetMetaData metaData = resultSet.getMetaData();
-
-      TableRow outputTableRow = new TableRow();
-
-      for (int i = 1; i <= metaData.getColumnCount(); i++) {
-        if (resultSet.getObject(i) == null) {
-          outputTableRow.set(metaData.getColumnName(i), resultSet.getObject(i));
-          continue;
-        }
-
-        /*
-         * DATE:      EPOCH MILLISECONDS -> yyyy-MM-dd
-         * DATETIME:  EPOCH MILLISECONDS -> yyyy-MM-dd hh:mm:ss.SSSSSS
-         * TIMESTAMP: EPOCH MILLISECONDS -> yyyy-MM-dd hh:mm:ss.SSSSSSXXX
-         *
-         * MySQL drivers have ColumnTypeName in all caps and postgres in small case
-         */
-        switch (metaData.getColumnTypeName(i).toLowerCase()) {
-          case "date":
-            outputTableRow.set(
-                metaData.getColumnName(i), dateFormatter.format(resultSet.getObject(i)));
-            break;
-          case "datetime":
-            outputTableRow.set(
```

Comment:
MySQL and SQL Server datetime value is not equal. I think should be define for each template